sdk/core/System.ClientModel/src/Pipeline/UserAgentPolicy.cs

```diff
@@ -73,18 +70,25 @@ public override async ValueTask ProcessAsync(PipelineMessage message, IReadOnlyL
         await ProcessNextAsync(message, pipeline, currentIndex).ConfigureAwait(false);
     }
 
+    /// <summary>
+    /// The properly formatted UserAgent string based on this <see cref="UserAgentPolicy"/> instance.
+    /// </summary>
+    public override string ToString() => _userAgent;
```

Comment:
I think using `ToString` here is a little unintuitive. How do you feel about instead exposing a read-only property called `UserAgentValue`?

Reply:
I agree it is somewhat controversial to have a displayable (non debug) string as the ToString value, but this is following the same pattern as TelemetryDetails.

Reply:
I don't have strong opinions one way or the other though, so happy to go with whatever the consensus is.

Reply:
Yeah good point.